WebRoot/messages/I18nMsg.properties

```diff
@@ -45,6 +45,8 @@ parseTime2 = (\\d)[\\.\\:\\s]?(\\d\\d)\\s*([ap])?
 parseTime3 = (\\d{1,2})(\\d*)\\s*([ap])?
 parseTimePMChars = pP
 
+
+miniCalendar = mini calendar
```

Comment:
this file is used specially for translations related to date/time formats, not sure if this label belongs here

Reply:
I think ZmMsg is good for this.